apps/server/src/controllers/targetControllers/createTargets.ts

```diff
@@ -12,7 +12,6 @@ import { createTargetSchema } from '../../utils/validators'
 
 export const createTarget = async (req: RequestExt, res: Response) => {
   const { _userId: userId } = req.body
-
   try {
     const requestData = createTargetSchema.safeParse(req.body)
```

Comment:
Why don't you use this approach & have everything in the try,catch block:

const parsedData = createTargetSchema.safeParse(req.body);

 const { _userId: userId, frequency, category, ...rest } = parsedData.data;

Reply:
I don't think you're right with this approach, sha.
Another way is to use the rest object and safeparse it in the zod schema.
`const { _userId: userId, ...rest} = req.body;`
`const requestData = createTargetSchema.safeParse(rest)`

This approach would be nicer, I think.